packages/adblocker/src/engine/engine.ts

```diff
@@ -326,34 +349,62 @@ export default class FilterEngine extends EventEmitter<
     // Subscription management: disabled by default
     this.lists = lists;
 
+    // Preprocessors
+    this.env = env;
+    this.preprocessors = new PreprocessorBucket({ env, preprocessors });
+
     // $csp=
-    this.csp = new NetworkFilterBucket({ config: this.config });
+    this.csp = new NetworkFilterBucket({
+      config: this.config,
+    });
     // $elemhide
     // $generichide
     // $specifichide
-    this.hideExceptions = new NetworkFilterBucket({ config: this.config });
+    this.hideExceptions = new NetworkFilterBucket({
```

Comment:
if possible, please don't change the code style - it makes this already big PR harder to read

Reply:
Sorry, I resolved this. I didn't notice while changing that part multiple parts.